x/leverage/client/tests/suite.go

```diff
@@ -17,8 +17,6 @@ import (
 type IntegrationTestSuite struct {
 	suite.Suite
 
-	abort bool // stop interdependent tests on the first error for clarity
-
```

Comment:
This wasn't working due to a misuse of `s.Run` - now it's not needed after the fix.